ruby/vampires.rb

```diff
@@ -1,3 +1,5 @@
+#random comment so I can make a pull request
+
 puts "How many employees will be processed today?"
 new_employees = gets.chomp.to_i
 

```

Comment:
My comments on the control flow are more of a design optimization suggestions.  Code be written in several ways and I understand that the directions may throw you off of this trail a bit.  The code runs as expected.  Im just adding context for a cleaner control flow :)